tests/embind/test_i64_binding.cpp

```diff
@@ -97,7 +98,7 @@ int main()
   ensure_js("v64.get(3) === -4n");
 
   execute_js("v64.push_back(1234n)");
-  ensure_js("v64.size() === 5");
+  ensure_js("v64.size() == 5");
```

Comment:
why `== 5` and not `=== 5n` (assuming that is the proper value)?

Reply:
I fixed this by fixing dynCall when the return value is `p` (pointer/size_t)